index.html

```diff
@@ -107,52 +158,71 @@ <h2>About</h2>
         </p>
         <button class="about-button" type="button" name="button"> Get my CV   <i class="fa-solid fa-download"></i></button>
       </section>
-      <section class="technologies-section languages">
-        <h3>Languages</h3>
-        <ul>
-          <li>JavaScript (ES5/ES6)</li>
-          <li>Ruby</li>
-          <li>HTML</li>
-          <li>CSS</li>
-        </ul>
-        <img src="images/icones/ic_language.png" alt="htag">
-      </section>
-      <section class="technologies-section frameworks">
-          <h3>Frameworks</h3>
+      <section class="technologies">
+        <section class="technologies-section languages">
+          <h3>Languages</h3>
           <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSpec</li>
-            <li>Capibara</li>
-            <li>Selenium</li>
+            <li>JavaScript (ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
           </ul>
-          <img src="images/icones/ic_framework.png" alt="htag">
-      </section>
-      <section class="technologies-section skills">
-        <h3>Skills</h3>
-        <ul>
-          <li>Codekit</li>
-          <li>Codepen</li>
-          <li>GitHub</li>
-          <li>GitLab</li>
-          <li>Terminal</li>
-        </ul>
-        <img src="images/icones/ic_skills.png" alt="htag">
+          <img src="images/icones/ic_language.png" alt="htag">
+        </section>
+        <section class="technologies-section frameworks">
+            <h3>Frameworks</h3>
+            <ul>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capibara</li>
+              <li>Selenium</li>
+            </ul>
+            <img src="images/icones/ic_framework.png" alt="htag">
+        </section>
+        <section class="technologies-section skills">
+          <h3>Skills</h3>
+          <ul>
+            <li>Codekit</li>
+            <li>Codepen</li>
+            <li>GitHub</li>
+            <li>GitLab</li>
+            <li>Terminal</li>
+          </ul>
+          <img src="images/icones/ic_skills.png" alt="htag">
+        </section>
+        <section class="technologies-section desk-deco">
+          <img class="shape" src="images/Shape.png" alt="shape">
+          <img class="imag-dot" src="images/image_dots.png" alt="htag">
+        </section>
```

Comment:
- [ ] In the mobile version, It would be nicer to add the gap between the `technologies` sections to improve your UI design and also to match the Figma design:
- Your design:
<img width="155" alt="image" src="https://user-images.githubusercontent.com/77190203/159908038-4a1b75ee-453f-490a-aa6f-aa636fad49ce.png">

- Figma design:
<img width="80" alt="image" src="https://user-images.githubusercontent.com/77190203/159908146-c0f83412-157a-41b1-a53d-accfbaa69603.png">